west.yml

```diff
@@ -116,7 +116,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: 6c223f7a18cf1b4d67c0390925c931c5332d303c
+      revision: pull/879/head
```

Comment:
The DNM will be removed by the bot automatically when you change this to a SHA. I merged the nrfxlib PR, please change this to the SHA and we can merge this manifest PR